AtReconstruction/AtCopyAuxTreeTask.cxx

```diff
@@ -0,0 +1,89 @@
+#include "AtCopyAuxTreeTask.h"
+
+#include "AtRawEvent.h"
+
+#include <FairLogger.h>
+#include <FairRootFileSink.h>
+#include <FairRootManager.h>
+#include <FairRunAna.h>
```

Comment:
/user/anthonya/attpcroot/AtReconstruction/AtCopyAuxTreeTask.cxx should remove these lines:
- #include <FairRootFileSink.h>  // lines 6-6
- #include <FairRunAna.h>  // lines 8-8
- #include <FairSink.h>  // lines 9-9